packages/experience-builder-sdk/src/ExperienceRoot.tsx

```diff
@@ -21,6 +21,13 @@ type ExperienceRootProps = {
 
 export const ExperienceRoot = ({ locale, experience, slug }: ExperienceRootProps) => {
   const [mode, setMode] = useState<InternalSDKMode>(() => {
+    if (!experience) {
+      if (window !== window.parent) {
```

Comment:
If I'm not mistaken, this check could also render in an SSR/ SSG context where `window` is not defined. We use this safeguard check (almost) everytime in the SDK when we need to access `window`.
```suggestion
      if (typeof window !== 'undefined' && window !== window.parent) {
```

Reply:
Good insight, thank you for the suggestion